packages/mobile/src/import/saga.test.ts

```diff
@@ -0,0 +1,63 @@
+import BigNumber from 'bignumber.js'
+import { expectSaga } from 'redux-saga-test-plan'
+import * as matchers from 'redux-saga-test-plan/matchers'
+import { call } from 'redux-saga/effects'
+import { setBackupCompleted } from 'src/account'
+import { showError } from 'src/alert/actions'
+import { ErrorMessages } from 'src/app/ErrorMessages'
+import { refreshAllBalances } from 'src/home/actions'
+import {
+  importBackupPhraseEmpty,
+  importBackupPhraseFailure,
+  importBackupPhraseSuccess,
+} from 'src/import/actions'
+import { importBackupPhraseSaga } from 'src/import/saga'
+import { redeemInviteSuccess } from 'src/invite/actions'
+import { waitWeb3LastBlock } from 'src/networkInfo/saga'
+import { fetchTokenBalanceWithRetry } from 'src/tokens/saga'
+import { setKey } from 'src/utils/keyStore'
+import { assignAccountFromPrivateKey } from 'src/web3/saga'
+import { mockAccount } from 'test/values'
+
+const mockPhraseValid =
+  'oil please secret math suffer mesh retreat prosper quit traffic special creek educate rate weasel wide swing crystal day swim frost oxygen course expire'
+const mockPhraseInvalid =
+  'oilsome pleasely secretment math suffer mesh retreat prosper quit traffic special creek educate rate weasel wide swing crystal day swim frost oxygen course expire'
+
+describe('Import wallet saga', () => {
+  it('imports a valid phrase', async () => {
+    // @ts-ignore
```

Comment:
What is it ignoring?

Reply:
It seems like a bug with expectSaga's typings for saga parameters